packages/cli/test/integration/credentials.controller.test.ts

```diff
@@ -1,33 +1,35 @@
-import * as testDb from './shared/testDb';
-import * as utils from './shared/utils/';
-import { randomCredentialPayload as payload } from './shared/random';
-
 import type { Credentials } from '@/requests';
 import type { User } from '@db/entities/User';
+import * as testDb from './shared/testDb';
```

Comment:
Would it make sense to import only the `truncate` function as it's the only one used?

Reply:
`testDb` is a lot smaller now. This won't have that much of an impact.

Reply:
but, I can definitely change this in part-2.